Sources/CryptomatorCloudAccess/Box/BoxCloudProvider.swift

```diff
@@ -579,7 +579,7 @@ public class BoxCloudProvider: CloudProvider {
 
 	private func convertStandardError(_ error: Error) -> Error {
 		switch error {
-		case let error as BoxAPIError where error.responseInfo.statusCode == 401:
+		case let error as BoxAPIError where error.responseInfo.statusCode == 401 || error.description.contains("Invalid refresh token"):
```

Comment:
I'd like this to be more specific. Something like:

```swift
case let error as BoxAPIError where (error.responseInfo.statusCode == 400 && error.responseInfo.rawBody.contains("invalid_grant")) || error.responseInfo.statusCode == 401:
```

I don't know if this works, but that's the most specific that I could find in [ResponseInfo](https://github.com/box/box-swift-sdk-gen/blob/main/Sources/Box/Errors/ResponseInfo.swift).

Reply:
Your suggestion worked! I just had to make two small adjustments. Will be in the next commit.